x/ccv/provider/types/keys.go

```diff
@@ -23,6 +23,9 @@ const (
 
 	// QuerierRoute is the querier route for IBC transfer
 	QuerierRoute = ModuleName
+
+	// Default validator set update ID
+	DefaultValsetUpdateID = 1
```

Comment:
From my understanding, this is the default starting value for the valset update ID on the provider. This counter would be set to something other than 1 if the provider halted, correct?